package.json

```
@@ -1,17 +1,24 @@
 {
   "name": "autobus",
-  "version": "0.1.6",
+  "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
   "directories": {
     "test": "test"
   },
+  "prettier": "prettier-config-surikaterna",
   "scripts": {
-    "test": "mocha test --recursive",
-    "coverage": "istanbul cover node_modules/mocha/bin/_mocha -- --recursive test -u exports -R spec",
-    "coverage-start": "istanbul cover node_modules/mocha/bin/_mocha test --recursive -- -u exports -R spec && start coverage/lcov-report/index.html",
-    "jshint": "jshint ./lib",
-    "jscs": "jscs -p google ./lib"
+    "test": "jest",
+    "test.watch": "jest --watch",
+    "compile": "rimraf lib && concurrently \"npm run compile.src\" \"npm run compile.types\"",
+    "compile.watch": "rimraf lib && concurrently \"npm run compile.src.watch\" \"npm run compile.types.watch\"",
+    "compile.src": "swc src --out-dir lib",
+    "compile.src.watch": "npm run compile.src -- --watch",
+    "compile.types": "tsc",
+    "compile.types.watch": "npm run compile.types -- --watch",
+    "prepare": "npm test && npm run compile",
+    "lint": "eslint src/**/*.ts test/**/*.ts"
```

There is no `test` dir anymore. 🙂 

```suggestion
    "lint": "eslint src/**/*.ts"
```